wurst/_handles/Unit.wurst

```diff
@@ -178,6 +178,15 @@ public function unit.getMaxHP() returns real
 public function unit.getMaxMana() returns real
 	return this.getState(UNIT_STATE_MAX_MANA)
 
+/** Get units current missing helth. */
+public function unit.getMissingHP() returns real
+	return this.getMaxHP()-this.getHP()
```

Comment:
add spacing around binary operators, see https://wurstlang.org/manual.html#coding-conventions

Reply:
I always need to do this in his code :((( so annoying xD